source/_docs/modules-plugins-known-issues.md

```diff
@@ -526,6 +526,18 @@ The list of [WordPress roles and capabilities](https://codex.wordpress.org/Roles
 * `edit_plugins`
 * `edit_themes`
 
+### [wp_filesystem->get_contents()](https://developer.wordpress.org/reference/classes/wp_filesystem_base/get_contents/){.external}
+
+**Issue**: The function `wp_filesystem->get_contents()` can fail when an environment is in Git mode (as Test and Live always are) because it is aware of filesystem-level permissions which are restricted in this mode.
+
+**Solution**: As described in [this StackExchange answer](https://wordpress.stackexchange.com/questions/166161/why-cant-the-wp-filesystem-api-read-googlefonts-json/166172#166172){.external}, for cases where file ownership doesn't matter this function could be replaced with `file_get_contents()`. This is true of most cases where the file in question is only being read, not written to.
+
+If the function cannot be replaced, you can create a workaround by adding to your `wp-config.php` file:
```

Comment:
What are the other implications of this workaround? I'm not sure it's something I'd recommend, as it has broader impact on WordPress' behavior.

Rather than including this workaround, you could simply leave this section as the "Issue" and "Solution.

Reply:
Another way of phrasing: including this workaround becomes the "official Pantheon supported solution". I don't think it's quite that, so better to leave unmentioned.